lib/pbench/test/functional/agent/cli/commands/tools/test_clear_tools.py

```diff
@@ -135,29 +137,29 @@ def test_clear_tools_test68(monkeypatch, agent_config, pbench_run, pbench_cfg):
     default_group = pbench_run / "tools-v1-default"
     default_group.mkdir(parents=True)
 
-    fubar5_group = default_group / "fubar5.example.com"
-    fubar5_group.mkdir(parents=True, exist_ok=True)
-    vmstat5_tool = fubar5_group / "vmstat"
-    vmstat5_tool.touch()
+    fubar1_group = default_group / "fubar1.example.com"
+    fubar1_group.mkdir(parents=True, exist_ok=True)
+    vmstat1_tool = fubar1_group / "vmstat"
+    vmstat1_tool.touch()
 
-    fubar6_group = default_group / "fubar6.example.com"
-    fubar6_group.mkdir(parents=True)
-    vmstat6_tool = fubar6_group / "vmstat"
-    vmstat6_tool.touch()
-    pidstat_tool = fubar6_group / "pidstat"
+    fubar2_group = default_group / "fubar2.example.com"
+    fubar2_group.mkdir(parents=True)
+    vmstat2_tool = fubar2_group / "vmstat"
+    vmstat2_tool.touch()
+    pidstat_tool = fubar2_group / "pidstat"
     pidstat_tool.touch()
-    turbostat_tool = fubar6_group / "turbostat"
+    turbostat_tool = fubar2_group / "turbostat"
     turbostat_tool.touch()
 
     command = [
         "pbench-clear-tools",
         "--name=vmstat",
-        "--remotes=fubar5.example.com,fubar6.example.com",
+        "--remotes=fubar1.example.com,fubar2.example.com",
     ]
     out, err, exitcode = pytest.helpers.capture(command)
     assert exitcode == 0
-    assert vmstat6_tool.exists() is False
-    assert vmstat5_tool.exists() is False
+    assert vmstat2_tool.exists() is False
+    assert vmstat1_tool.exists() is False
```

Comment:
> its a very small change which doesn't change any logic per se but just some variable names

Yes, and "every snowflake in an avalanche pleads 'not guilty'".  😆

> the code does change [...] and so it warrants changes in our tests as well.

True, but the fact that we have to change the tests to meet the new code behavior doesn't mean that should make other changes to the tests which are not relevant to the new code behavior.  🙂

Making non-functional or non-logic-changing modifications is not necessarily a bad thing -- it is important for the readability and supportability of the code -- but mixing those modifications with changes which _are_ functional _**is**_ a bad thing, because it makes it harder to see what functional changes were made (and, in the event that we need to back out changes, it's harder because there is more change, some of which we might want to keep).

We can "get away with" mixing functional and non-functional changes in the same PR if they are carefully segregated into separate commits.  However, preserving (or restoring) that segregation is cumbersome if the PR generates requests for changes.  So, it's best (if a bit awkward) to keep the two types of changes to separate PRs:  the reviews are easier and faster, responding to review feedback is straightforward, the PR branches can be easily squashed, and later forensic analysis of the code (including backing stuff out) is much more straightforward.  (And, it's just plain less likely to result in bugs.)

But, I know, the temptation to "just fix that up while I'm here" is _very_ strong....  😁